cmake/config.cmake

```diff
@@ -40,14 +40,11 @@ set(BUILD_CPP_TEST OFF)
 # Whether to enable OpenMP.
 set(USE_OPENMP ON)
 
-# Whether to enable Intel's avx optimized kernel.
-set(USE_AVX ON)
-
 # Whether to build PyTorch plugins.
 set(BUILD_TORCH ON)
 
 # Whether to enable CUDA kernels compiled with TVM.
 set(USE_TVM OFF)
 
-# Whether to enable fp16 to support mixed precision training.
-set(USE_FP16 OFF)
```

Comment:
pls remove `USE_FP16` from `//docs/source/guide_ko/mixed_precision.rst` as well

Reply:
Removed. I copy the ko doc to google translate and remove paragraphs about building from source with USE_FP16.